dev-tools/packaging/packages.yml

```diff
@@ -157,6 +157,19 @@ shared:
         template: '{{ elastic_beats_dir }}/dev-tools/packaging/templates/windows/uninstall-service.ps1.tmpl'
         mode: 0755
 
+  - &docker_spec
+    <<: *binary_spec
+    extra_vars:
+      from: 'centos:7'
+      user: '{{ .BeatName }}'
+      repository: 'docker.elastic.co/beats'
```

Comment:
can we add a param for this?

Reply:
As this is specific of docker packaging I left it as an extra var (like the specific extra vars for DMG builds). This extra var is optional, and can be set from `packages.yml` or overriden with `spec.ExtraVar()` from the magefile.

I am open to move it to other kind of parametrization though.

Reply:
I have added 8e6a32356 so it is not set by default on community beats.